app.js

```diff
@@ -1,14 +1,32 @@
 const express = require ('express');
-const dotenv = require ('dotenv');
+const dotenv = require ('dotenv'); //what is this
```

Comment:
https://www.npmjs.com/package/dotenv

_Dotenv is a zero-dependency module that loads environment variables from a .env file into process.env. _